tests/network-tests/src/fixtures/content/activeVideoCounters.ts

```diff
@@ -0,0 +1,383 @@
+
+    this.debug('Adding storage bag to bucket')
+    await this.api.updateStorageBucketsForBag(
+      storageLeader.role_account_id.toString(),
+      channelIds[0].toString(),
+      [storageBucketId],
+    )
+
+    this.debug('Accepting content to storage bag')
+    //const bagId = `dynamic:channel:${channelIds[0]}`
+    const allAssetIds = videosData.map((item) => item.assetContentIds).flat()
+    await this.api.acceptPendingDataObjects(
+      storageGroupWorker.keyringPair.address,
+      storageGroupWorkerId,
+      storageBucketId,
+      //bagId,
+      channelIds[0].toString(),
+      allAssetIds,
+    )
+
+    // check channel and categories con are counted as active
+
+    this.debug('Checking channels active video counters')
+    await this.assertCounterMatch('channels', channelIds[0], videoCount)
+
+    this.debug('Checking channel categories active video counters')
+    await this.assertCounterMatch('channelCategories', channelCategoryIds[0], videoCount)
+
+    this.debug('Checking video categories active video counters')
+    await this.assertCounterMatch('videoCategories', videoCategoryIds[0], videoCount)
+
+    // move channel to different channel category and video to different videoCategory
+
+    const oneMovedVideoCount = 1
+    this.debug('Move channel to different channel category')
+    await this.cli.updateChannel(channelIds[0], {
+      category: channelCategoryIds[1], // move from category 1 to category 2
+    })
+
+    this.debug('Move video to different video category')
+    await this.cli.updateVideo(videosData[0].videoId, {
+      category: videoCategoryIds[1], // move from category 1 to category 2
+    })
+
+    // check counters of channel category and video category with newly moved in video/channel
+
+    this.debug('Checking channel categories active video counters (2)')
+    await this.assertCounterMatch('channelCategories', channelCategoryIds[1], videoCount)
+
+    this.debug('Checking video categories active video counters (2)')
+    await this.assertCounterMatch('videoCategories', videoCategoryIds[1], oneMovedVideoCount)
+
+    /** Giza doesn't support changing channels - uncoment this on later releases where it's \\\
+
+    // move one video to another channel
+
+    this.debug('Move video to different channel')
+    await this.cli.updateVideo(videosData[0].videoId, {
+      channel: channelIds[1], // move from channel 1 to channel 2
+    })
+
+    // check counter of channel with newly moved video
+
+    this.debug('Checking channels active video counters (2)')
+    await this.assertCounterMatch('channels', channelIds[0], videoCount - oneMovedVideoCount)
+    await this.assertCounterMatch('channels', channelIds[1], oneMovedVideoCount)
+
+    // end
+    */
+
+    this.debug('Done')
+  }
+
+  /**
+    Prepares storage requisites.
+  */
+  private async prepareAssetStorage(storageLeader: Worker, storageGroupWorker: IMember) {
```

Comment:
This is handled by https://github.com/Joystream/joystream/blob/giza_staging/tests/network-tests/src/flows/storagev2/initStorage.ts on `giza_staging`, which is part of the scenario executed during `run-all-tests`: https://github.com/Joystream/joystream/blob/giza_staging/tests/network-tests/src/scenarios/combined.ts

Reply:
The function has been removed and the existing `initStorage` flow is used instead.